SWLOR.Game.Server/Feature/ChatCommandDefinition/DMChatCommand.cs

```diff
@@ -51,6 +51,7 @@ public class DMChatCommand: IChatCommandListDefinition
             Notes();
             CreatureManager();
             Broadcast();
+            SetScale();
```

Comment:
I'd recommend doing a /getScale command here too. It may be helpful to retrieve that information if possible.